src/backend/didConfiguration/didConfigResource.ts

```diff
@@ -31,40 +27,43 @@ async function attestDomainLinkage() {
     configuration.did,
   );
 
-  const requestForAttestation = RequestForAttestation.fromClaim(claim);
+  const credential = Credential.fromClaim(claim);
 
   const { fullDid } = await fullDidPromise;
 
-  const attestationKey = fullDid.attestationKey;
+  const attestationKey = fullDid.assertionMethod?.[0];
   if (!attestationKey) {
     throw new Error('The attestation key is not defined?!?');
   }
 
-  const { signature, keyUri } = await fullDid.signPayload(
-    Crypto.coToUInt8(requestForAttestation.rootHash),
-    assertionKeystore,
+  // TODO: does that do anything fancy or can we just use createPresentation?
+  const { signature, keyUri } = await signPayload(
+    fullDid,
+    Crypto.coToUInt8(credential.rootHash),
+    assertionSigner,
     attestationKey.id,
   );
```

Comment:
are we doing this bc we want to use the `assertionMethod` key instead of the `authentication` key?

Reply:
Yes, it has to assertionMethod according to the well-known DID specification.

Reply:
in that case we should be able to use createPresentation but just pass it the assertionSigner